recipes/xyzspaces/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "xyzspaces" %}
+{% set version = "0.3.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/heremaps/xyz-spaces-python/archive/{{ version }}.tar.gz
+  sha256: 060f023a9711d0d6abe1381ec2723ee8ec0c6551f24a413098295bf8b58121ed
+
+build:
+  number: 0
+  skip: True  # [py<36]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
```

Comment:
You need to add everything in the `setup.py`'s `install_requires`, which looks to be read out of:

https://github.com/heremaps/xyz-spaces-python/blob/master/requirements.txt

I'm guessing the issue with `fiona` not being on the list, but being required is that it is a dependency of one of the listed dependencies, so it gets picked up. Just a guess though